.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+	"image": "mcr.microsoft.com/vscode/devcontainers/python:3",
+
+	// Configure tool-specific properties.
+	"customizations": {
+		// Configure properties specific to VS Code.
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				// Settings for the python extension.
+				"python.defaultInterpreterPath": "${workspaceFolder}/.venv",
+				"python.terminal.activateEnvInCurrentTerminal": true,
+				"python.linting.enabled": true,
+				"python.linting.pylintEnabled": true
+
+			},
+			// Installs extensions on container create.
+			"extensions": [
+				"ms-python.python",
+				"ms-python.vscode-pylance"
+			]
```

Comment:
This installs the python extension and uses it to set the default interpreter and activate the venv.

Because the extension is still loading at initial creation, the venv doesn't activate on the initial terminal. It does activate on any new terminals or upon reconnecting to a codespace. I updated the welcome text and contributing guide accordingly.